### ISSUE TITLE
Impossible to create a new node on keyboard

### ISSUE THREAD

**Issue by abdermaiza** [xyflow/xyflow#5620]
### What platform were you using when you found the bug?

- React Flow / Svelte Flow version: 
- Browser and version: Chrome
- OS and version: Windows 10 Pro

### Describe the Bug

Hello,

When using you keyboard on this component:
https://reactflow.dev/examples/nodes/add-node-on-edge-drop

<img width="412" height="196" alt="Image" src="https://github.com/user-attachments/assets/35427921-60e5-474b-a0d1-d133c94aebec" />

It's impossible to reach that circle to create a new node with your keyboard, via Tab.

Thanks.

### Steps to reproduce the bug or issue

1. Go to "https://reactflow.dev/examples/nodes/add-node-on-edge-drop"
2. Try to interact with the node with your keybard
3. Reaching the drop (to create a new node) seems impossible

### Expected behavior

As a keyboard user or any other device like screen readers, I should be able to add a new node only with it, without my mouse.

### Screenshots or Videos

<img width="412" height="196" alt="Image" src="https://github.com/user-attachments/assets/1a4aea29-cf55-406d-a1bf-911a52122130" />

**Comment by moklick** [comment]
This is a great feature that we would love to see! It would break the existing logic where Tab is used to select nodes, but this could be an improvement or an advanced keyboard mode for React Flow 13. I will leave it open and add the "next major" label.
